README.md

```diff
@@ -4,6 +4,10 @@
 
 ## [Documentation](https://docs.rs/halo2)
 
+This crate currently has a Minimum Supported Rust Version of 1.49.0. When const generics
```

Comment:
If we _are_ deciding to pin a MSRV now, then we should both make this change in CI, and add a `rust-toolchain` file.